Sources/StreamVideoSwiftUI/CallingViews/MicrophoneCheckView.swift

```diff
@@ -97,19 +97,8 @@ public struct AudioVolumeIndicator: View {
         return levels
     }
     
-    private func height(for decibel: Float) -> CGFloat {
-        let value = abs(decibel)
-        let divider = (maxValue - minValue) / 2
-        if divider == 0 {
-            return 0
-        }
-        let ratio = value / divider
-        var height: CGFloat
-        if decibel > 0 {
-            height = CGFloat(ratio * maxHeight)
-        } else {
-            height = CGFloat(maxHeight - ratio * maxHeight)
-        }
+    private func height(for value: Float) -> CGFloat {
```

Comment:
why the max 0.5?

Reply:
This is because if the mic audioLevel is 0 and we return a `0` height then the sound bar won't be visible. With the `0.5` when there is "no sound" we display the sound bar as a dot(of height 0.5 pixels * scale of screen)